docs/source/en/generation_strategies.md

```diff
@@ -173,6 +173,46 @@ your screen, one word at a time:
 An increasing sequence: one, two, three, four, five, six, seven, eight, nine, ten, eleven,
 ```
 
+
+## KV Cache Quantization
+
+The `generate()` supports caching keys and values to enhance efficieny and avoid re-computations. However the key and value
+cache can occupy a large portion of memory becoming a bottleneck for long-context generatiomemory, especially for Large Language Models.
+Quantizing the cache when using `generate()` can significantly reduce memory demands in the cost of speed. 
+
+KV Cache quantization in `transformers` is largely inspired by the paper [KIVI: A Tuning-Free Asymmetric 2bit Quantization for KV Cache]
+(https://arxiv.org/abs/2402.02750). For more information on the inner workings see the paper.
+
+To enable quantization of the key-value cache, one needs to indicate `cache_immplementation=True` in the `generation_config`.
+Quantization related arguments should be passed to the `generation_config` either as a `dict` or an instance of a `CacheConfig` class.
```

Comment:
Oops, should be cache_implementation="quantized". The docs are misleading 😄 

But yes, "cache_config" would be perfect instead of passing both and can go to another PR I guess

Reply:
Yes! `cache_implementation="quantized"` is good enough for now.

In the long run, we will probably gravitate toward the cache config: each new cache has new flags. A problem for a future PR